components/browser/state/src/main/java/mozilla/components/browser/state/state/ContainerState.kt

```diff
@@ -2,7 +2,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package mozilla.components.feature.containers
+package mozilla.components.browser.state.state
+
+/**
+ * Value type that represents the state of a container.
+ *
+ * @property contextId The session context ID also known as cookie store ID for the container.
+ * @property name Name of the container.
+ * @property color The color for the container. This can be shown in tabs belonging to this container.
+ * @property icon The name of an icon for the container.
+ */
+data class ContainerState(
+    override val contextId: String,
+    override val name: String,
+    override val color: String,
+    override val icon: String
```

Comment:
I'm also wondering about the type here, what does "name of an icon" actually mean? How can I display it? Could this be a Bitmap? Or a Drawable? Or a file path? How do I currently turn this name into something I can display?

Reply:
Changed to Drawable